devtools.js

```diff
@@ -0,0 +1,58 @@
+var createStore = require('.')
+
+function devtools (modules) {
+  var extension =
+    window.__REDUX_DEVTOOLS_EXTENSION__ ||
+    window.top.__REDUX_DEVTOOLS_EXTENSION__
+
+  if (!extension) {
+    console.warn('Please install Redux devtools extension')
+    return createStore(modules)
+  }
+
+  var ReduxTool = extension.connect()
+
+  var store = createStore(modules.concat(createLogger(ReduxTool)))
```

Comment:
Maybe we can do the whole `devtools` function as a module (as we did for `./logger.js`) instead of `createStore` fork?

Reply:
Will look into that